mlir/utils/jenkins/premerge-checks.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python3
```

Comment:
Desirably, add a comment before each function referring to where to find it in the `premerge-checks` repo. Point them to `premerge_checks.py` / `clang_format_report.py` / `clang-tidy_report.py`.

Also, add a llvm license header section with a brief description of this file, and document the bash command to run for a individual file.